solr/contrib/ltr/example/train_and_upload_demo_model.py

```diff
@@ -130,33 +126,31 @@ def main(argv=None):
     with open(options.configFile) as configFile:
         config = json.load(configFile)
 
-        print "Deploying feature space to Solr"
+        print "Uploading feature space to Solr"
         setupSolr(config)
 
         print "Generating feature extraction Solr queries"
         reRankQueries = generateQueries(config)
 
         print "Extracting features"
-        output = open(config["trainingFile"],"w")
         fvGenerator = generateTrainingData(reRankQueries, config);
-        formatter = libSvmFormatter.LibSvmFormatter();
+        formatter = libsvm_formatter.LibSvmFormatter();
         formatter.processQueryDocFeatureVector(fvGenerator,config["trainingFile"]);
 
         print "Training ranksvm model"
```

Comment:
Thought about it, but since this is a demo script for people that dont know how to train a model, I wanted it to be as simple as possible without extra dependencies.